### ISSUE TITLE
Linux version doesn't start/work

### ISSUE THREAD

**Issue by bl00dy1837** [abc013/WarriorsSnuggery#39]
```
publish_linux_x64$ ./WarriorsSnuggery 
Loading...Ouch! An error occurred.
For more details, please check out the logs (located in 'WarriorsSnuggery/logs/').
Please report the issue to https://github.com/abc013/WarriorsSnuggery or contact abc013.
Thank you!
Unhandled exception. System.IO.FileNotFoundException: Unable to find the specified file.
   at System.Drawing.Text.PrivateFontCollection.AddFontFile(String filename)
   at WarriorsSnuggery.Graphics.FontManager.Load() in C:\Users\Meredith McKay\source\repos\WarriorsSnuggery\WarriorsSnuggery.Game\Graphics\Font\FontManager.cs:line 19
   at WarriorsSnuggery.Window.OnLoad() in C:\Users\Meredith McKay\source\repos\WarriorsSnuggery\WarriorsSnuggery.Game\Window.cs:line 137
   at OpenTK.Windowing.Desktop.GameWindow.Run()
   at WarriorsSnuggery.Program.run(String[] args) in C:\Users\Meredith McKay\source\repos\WarriorsSnuggery\WarriorsSnuggery.Game\Program.cs:line 101
   at WarriorsSnuggery.Program.Main(String[] args) in C:\Users\Meredith McKay\source\repos\WarriorsSnuggery\WarriorsSnuggery.Game\Program.cs:line 38
Aborted (core dumped)
```

**Comment by abc013** [comment]
Hey, sorry, the notification about your issue must have gone lost over the past few weeks. I am very sorry for the delay! It was kind of expected that the linux version won't work, but since I don't have any way to test it on my own, thank you for trying and opening an issue!

The problem is located in [FontManager.cs#L19](https://github.com/abc013/WarriorsSnuggery/blob/master/WarriorsSnuggery.Game/Graphics/Font/FontManager.cs#L19) and [FontManager.cs#L22](https://github.com/abc013/WarriorsSnuggery/blob/master/WarriorsSnuggery.Game/Graphics/Font/FontManager.cs#L22), where the windows directory char separator `/` is still used, I must've overlooked it while doing the changes and will fix it tomorrow.

Are you able to compile via source code?

**Comment by abc013** [comment]
Okay, so I went through some more possible problematic locations and actually found a bunch of other occassions where i missed the char separator change. It should be fixed via https://github.com/abc013/WarriorsSnuggery/commit/b4c6035d5730149f03ddc82c05636307dcf6e05a.
I'll leave this issue open since the commit probably won't fix the whole thing.

**Comment by abc013** [comment]
Okay, so I actually found another issue, fixed it and, with some help, managed to run WS on my own ubuntu virtual machine! 🎉 (It doesn't look neat yet but I'll look further into it.) 
It should work now if you compile from source.
If it crashes, it's probably because there are two packages missing: `libgdiplus` and `libopenal` (I used `libopenal-dev` since `libopenal` itself doesn't exist for whatever reason), which can be installed via `apt-get`.
